ecs-cli/modules/commands/flags.go

```diff
@@ -105,6 +107,12 @@ func OptionalRegionAndProfileFlags() []cli.Flag {
 				"[Optional] Specifies the name of the ECS profle configuration to use. Defaults to the default profile configuration.",
 			),
 		},
+		cli.StringFlag{
+			Name: AWSProfileNameFlag,
+			Usage: fmt.Sprintf(
+				"[Optional] Use your AWS credentials from an existing named profile in ~/.aws/credentials.",
```

Comment:
Uses*

Reply:
As per our IRL discussion, maybe add where the CLI looks for credentials by default if not set in the config file.

Reply:
I agree that the order of search for creds should be better documented. My proposed docs for all the configuration changes include info on that. I don't think this is the best place to document that though. I am thinking of adding an info print statement when the flags are present but they are being overridden by the environment variables (since this is the case which is most likely to confuse users).